src/styles/components/_card.scss

```diff
@@ -25,7 +25,10 @@
 
 .w-card:hover,
 .w-card:focus,
-.w-card:active {
+.w-card:active,
+.w-card--link:hover,
```

Comment:
I _think_ you might be able to get rid of these `w-card--link` styles and instead add a `w-card:focus-within` selector. I also left some comments above about moving the `w-card--link` anchor around.

Reply:
oh I was mistaken. I meant :focus-within should be used on card.scss not the chip :P